ForceSpotifySync_Linux/ForceSpotifySync.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Set script parameters
+echo "Searching in for collection caches in: "$SPOTIFY_PATH""
+SPOTIFY_PATH="$HOME"/.cache/spotify/Users/
+COLLECTIONS="$(find "$SPOTIFY_PATH" -name 'collection*')"
```

Comment:
```suggestion
SPOTIFY_PATH="$HOME"/.cache/spotify/Users/
COLLECTIONS="$(find "$SPOTIFY_PATH" -name 'collection*')"
echo "Searching in for collection caches in: "$SPOTIFY_PATH""
```

Shouldn't `SPOTIFY_PATH` variable be declared before `echo` ?